crates/conformance-test-utils/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "conformance-test-utils"
+version = { workspace = true }
+edition = { workspace = true }
+description = { workspace = true }
+homepage = { workspace = true }
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4.2", features = [ "derive" ] }
+starknet = "0.4.0"
```

Comment:
not from workspace?

Reply:
Gotcha, will use the one from workspace 👍